src/main/java/org/verdictdb/connection/InMemoryAggregate.java

```diff
@@ -80,13 +85,13 @@ public static void createTable(DbmsQueryResult dbmsQueryResult, String tableName
     statement.executeBatch();
```

Comment:
Please pull from the 'yongjoo' branch to apply the code that close these statements.